src/rrules/foreigncall.jl

```diff
@@ -156,6 +156,24 @@ function rrule!!(
     return y, NoPullback(ntuple(_ -> NoRData(), 9))
 end
 
+@static if VERSION >= v"1.11-rc4"
+    function rrule!!(
+        ::CoDual{typeof(_foreigncall_)},
+        ::CoDual{Val{:jl_genericmemory_copy}},
+        ::CoDual,
+        ::CoDual{Tuple{Val{Any}}},
+        ::CoDual{Val{0}},
+        ::CoDual{Val{:ccall}},
+        x::CoDual{<:Memory},
+    )
+        y = CoDual(
+            ccall(:jl_genericmemory_copy, Ref{Memory}, (Any,), primal(x)),
+            ccall(:jl_genericmemory_copy, Ref{Memory}, (Any,), tangent(x)),
+        )
```

Comment:
```suggestion
        y = CoDual(primal(x), tangent(x))
```
It's probably fine to simplify this to simply `copy`ing the memory, since the implementation of `copy` is a one-liner which calls `:jl_genericmemory_copy`.